tests/e2e/tests/build/rebuild-types.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  killAllProcesses,
+  waitForAnyProcessOutputToMatch,
+  execAndWaitForOutputToMatch,
+} from '../../utils/process';
+import { writeFile, prependToFile } from '../../utils/fs';
+import {getGlobalVariable} from '../../utils/env';
+
+
+const successRe = /webpack: Compiled successfully/;
+
+export default async function() {
+  if (process.platform.startsWith('win')) {
+    return Promise.resolve();
```

Comment:
Should be able to just `return;` here.  An async function will return a resolved promise with the returned value.

Reply:
You’re right. Thanks